bigquery/README.md

```diff
@@ -25,6 +25,27 @@ subdirectory.
 Edit ohdsi.yaml and update the zone or dataset names if desired.  Documentation
 is available in ohdsi.jinja.schema.
 
+If you want to reuse existing datasets for the deployment instead of creating
+new ones, you need to grant a few permissions that are needed for the VM to
+access the datasets. Say the project that hosts your datasets is A, and the
+project you want to deploy to is B (A and B can be the same):
+
+- Read permission on `cdm` dataset (or whatever name you change it to)
+  * Look up the default compute engine service account for B, it should look
+    like `123456789000-compute@developer.gserviceaccount.com`. Select the dataset
+    and click the caret on top right corner and choose "Share dataset" in the
+    dropdown list, input the service account and click "Add" then "Save changes"
+- Edit permission on `ohdsi` and `temp`
```

Comment:
It probably makes sense to always have local copies of these in B.  Only the cdm dataset really needs to be shared.

Reply:
I wasn't sure if we wanted to keep result/temp datasets locally, because from my understanding the result dataset will have the same data, so running the R script for each deployment seems redundant.

I tried to update the config to keep result/temp datasets in the local project, and it seems to work, except that I need to update the jdbc driver to support drop table `project`.dataset.table for runAchilles.R to work. Those changes are in other commits.

Reply:
Ignore last part from previous comment, the setup works. I am going to prepare an image release after this is submitted.